lib/CMakeLists.txt

```diff
@@ -76,8 +76,13 @@ target_link_libraries(scion PRIVATE ${DEPENDENCY_INSTALL_DIR}/lib/libnghttp2.a)
 # Link protobuf-c
 target_link_libraries(scion PRIVATE ${DEPENDENCY_INSTALL_DIR}/lib/libprotobuf-c.a)
 
+# Link zlib for gzip decompression
+find_package(ZLIB REQUIRED)
```

Comment:
This assumes that zlib installed on every system. We should probably document this, right?